tests/cypress/tests-e2e/compareSlider.cy.js

```diff
@@ -0,0 +1,445 @@
+                cy.get('[data-cy="compare_slider"]').trigger('mousemove', {
+                    clientX: Math.abs(x + 20),
+                    clientY: 100,
+                })
+                cy.get('[data-cy="compare_slider"]').trigger('mouseup', { force: true })
+            }
+            it('moves when we click on it and drag the mouse', () => {
+                cy.goToMapView(
+                    {
+                        layers: 'test-1.wms.layer',
+                        compare_ratio: '0.5',
+                    },
+                    true
+                )
+                // initial slider position is width / 2 -20
+                cy.get('[data-cy="compare_slider"]').then((slider) => {
+                    cy.readStoreValue('state.ui.width').then((width) => {
+                        cy.wrap(slider.position()['left']).should('eq', width / 2 - 20)
+                    })
+                })
+
+                cy.readStoreValue('state.ui.compareRatio').then((compareRatio) => {
+                    expect(compareRatio).to.eq(0.5)
+                })
+                moveSlider(73)
+                cy.get('[data-cy="compare_slider"]').then((slider) => {
+                    cy.wrap(slider.position()['left']).should('be.gte', 72.8)
+                    cy.wrap(slider.position()['left']).should('be.lte', 73.2)
+                })
+            })
+            it('refuses to get out of bounds when we move around', () => {
+                cy.goToMapView(
+                    {
+                        layers: 'test-1.wms.layer',
+                        compare_ratio: '0.5',
+                    },
+                    true
+                )
+
+                moveSlider(-10)
+                cy.get('[data-cy="compare_slider"]').then((slider) => {
+                    //an epsilon here : we noticed that it was at -5.99999
+                    cy.wrap(slider.position()['left']).should('be.gte', -6.01)
+                })
+
+                cy.readStoreValue('state.ui.width').then((width) => {
+                    moveSlider(width - 1)
+                })
+
+                cy.get('[data-cy="compare_slider"]').then((slider) => {
+                    cy.readStoreValue('state.ui.width').then((width) => {
+                        cy.wrap(slider.position()['left']).should('be.lte', width - 33.99)
+                    })
+                })
+            })
+        })
+        /*
+            in the current implementation, cutting a layer doesn't stop it from being selected
+            (this is the exact same implementation here as in the old mapviewer). While the features
+            are not visible, they are still selectable. Once we have a compare Slider which stops features
+            from being selected when cut, we can reactivate the skipped tests here
+        */
+        context.skip('Cutting layers', () => {
+            it.skip('cuts from the correct spot until the end of the map, and only from the top layer', () => {
```

Comment:
I think it would be better to only keep the `.skip` from the parent context, and not skip children. It will be easier in the future to re-activate the whole block this way.